src/gpgpu-sim/gpu-sim.cc

```diff
@@ -93,6 +93,7 @@ tr1_hash_map<new_addr_type, unsigned> address_random_interleaving;
 #define L2 0x02
 #define DRAM 0x04
 #define ICNT 0x08
+#define GMMU 0x10
```

Comment:
Minor nitpicking: Can you rewrite this in the form of `1 << 1, 1 << 2`? Just to keep things a bit cleaner.

Reply:
If we were to adopt the form "1 << n", we better wrap them in a pair of parentheses at macro definition. The C++ shift operators have very low precedence.